### PR TITLE
Add December 2024 newsletter draft

### LINKED ISSUE #807: Newsletter 2024-12

### PR DESCRIPTION
**Pull request by tdjames1**
Leaving this in draft until @Romain-Thomas-Shef has finalised dates and times for FAIR2RS semester 2.

Closes #807

### CODE REVIEW

File: _newsletters/2024-12-06-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+
+The [FAIR<sup>2</sup> for research software training programme](https://rse.shef.ac.uk/training/fair4rs/) has been developed by the RSE, Data Analytics Service and Library teams and launched in October. The full syllabus is now live on [MyDevelopment](https://mydevelopment.csod.com/ui/lms-learning-details/app/curriculum/06573f55-12e4-44f1-be4c-fac5ea9d6851) (requires login). The programme will be continuing over the next few months, with the following courses now scheduled:
+
+- 2024-12-09 &amp; 2024-12-10, 9.30am \- 1.00pm [Software Design](https://rse.shef.ac.uk/training/fair4rs/#software-design)
+- 2025-01-20 &amp; 2025-01-21, 9.30am \- 1.00pm [Git, GitHub and GitKraken - From Zero to Hero](https://rse.shef.ac.uk/training/fair4rs/#version-control)
+- 2025-02-06, 2.00pm - 5.00pm [Testing and Continuous Integration](https://rse.shef.ac.uk/training/fair4rs/#testing-and-continuous-integration)
```
Comment (ns-rse) [inline _newsletters/2024-12-06-newsletter.md]: To align with other rows where this has been escaped, but both render fine on Markdown Live Preview

```suggestion
- 2025-02-06, 2.00pm \- 5.00pm [Testing and Continuous Integration](https://rse.shef.ac.uk/training/fair4rs/#testing-and-continuous-integration)
```

File: _newsletters/2024-12-06-newsletter.md [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+- 2025-02-17 &amp; 2025-02-18, 9.30am \- 1.00pm [Git Collaboration](https://rse.shef.ac.uk/training/fair4rs/#version-control)
+- 2025-03-06, 9.30am \- 1.00pm [Documentation](https://rse.shef.ac.uk/training/fair4rs/#documentation)
+- 2025-03-24, 9.30am \- 1.00pm [Reproducible computational environments](https://rse.shef.ac.uk/training/fair4rs/#reproducible-computational-environments)
+- 2025-04-11, 2.00pm - 5.00pm [Packaging](https://rse.shef.ac.uk/training/fair4rs/#packaging)
```
Comment (ns-rse) [inline _newsletters/2024-12-06-newsletter.md]: To align with other rows where this has been escaped, but both render fine on Markdown Live Preview

```suggestion
- 2025-04-11, 2.00pm \- 5.00pm [Packaging](https://rse.shef.ac.uk/training/fair4rs/#packaging)
```

**Review by ns-rse — APPROVED** [review]
Suggestions for consistency in the Markdown but they don't impact the rendering (not seen use of escapes before `\-`).

If we were linting (#701) there would be complaints about line length but that doesn't look like it will ever take hold so not going to suggest reformatting that aspect.

**Comment by ns-rse** [comment]
Just checked and it doesn't yet appear possible to register for the [Git Collaboration course via myDevelopment](https://mydevelopment.csod.com/ui/lms-learning-details/app/event/6197989b-bef2-4352-a14b-02c4f45bbca6).

All the other courses this semester have dates listed.

- :white_check_mark: [git &amp; GitHub through GitKraken - from Zero to Hero!](https://mydevelopment.csod.com/ui/lms-learning-details/app/event/d4d09f67-097b-4451-8ddb-86cb90636c06)
- :white_check_mark: [Design you code...(and write less of it)](https://mydevelopment.csod.com/ui/lms-learning-details/app/event/d3c2cca5-d894-44b6-a79b-e93dea7a7c94)
- :white_check_mark: [Testing and Continuous Integration Course](https://mydevelopment.csod.com/ui/lms-learning-details/app/event/460c53c5-40e4-4c97-ab4f-144213544026)
- :white_check_mark: [Software Documentation](https://mydevelopment.csod.com/ui/lms-learning-details/app/event/7c81f9fd-656f-4ae9-ba48-bc83afe81cc2)
- :white_check_mark: [Reproducible Computational Environment](https://mydevelopment.csod.com/ui/lms-learning-details/app/event/e30c407d-c97d-44a0-b3fd-4e87f95c4789)
- :white_check_mark: [Software Packaging](https://mydevelopment.csod.com/ui/lms-learning-details/app/event/0cbafddd-4c31-4e8a-bce6-a12e895a4acf)

I checked in a second browser to ensure its not a caching issue.

@Romain-Thomas-Shef : Any idea why registration might not be available for this one course?

**Comment by tdjames1** [comment]
> Just checked and it doesn't yet appear possible to register for the [Git Collaboration course via myDevelopment](https://mydevelopment.csod.com/ui/lms-learning-details/app/event/6197989b-bef2-4352-a14b-02c4f45bbca6).

@Romain-Thomas-Shef is working on this, should be done later today.

**Pull request merged**